packages/paste-website/src/components/customization-landing-page/LandingPageDesigner/TextToken.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {useUIDSeed} from '@twilio-paste/uid-library';
+import {Input} from '@twilio-paste/input';
+import {Label} from '@twilio-paste/label';
+import type {GenericTokensShape} from '@twilio-paste/design-tokens/types/GenericTokensShape';
+import type {DesignerContextProps} from './DesignerContext';
+
+type TextTokenInputProps = {
+  bucket: keyof GenericTokensShape;
+  labelText: string;
+  tokenName: string;
+  tokenValue: string;
+  onChange: DesignerContextProps['updateToken'];
+};
+
+// TODO
+export const TextTokenInput: React.FC<TextTokenInputProps> = ({bucket, labelText, tokenName, tokenValue, onChange}) => {
+  const seed = useUIDSeed();
+
+  return (
+    <Box key={tokenName} marginBottom="space60">
+      <Label htmlFor={seed(tokenName)}>{labelText}</Label>
+      <Input
+        id={seed(tokenName)}
+        type="text"
+        value={tokenValue}
+        onChange={(event) => {
+          const {value} = event.target;
+          // TODO validate
```

Comment:
This is probably fine as is for a demo, but maybe there's a quick win to just validate that value exists as a token key?

Reply:
This is more validating if it's a valid value for the type of token you are editing. Is this a valid color? Is this a valid box shadow etc